api/api_types.go

```diff
@@ -13,20 +13,20 @@ import (
 )
 
 type Organization struct {
-	OrganizationID types.HexBytes      `json:"organizationID,omitempty"`
+	OrganizationID types.HexBytes      `json:"organizationID,omitempty" swaggertype:"string"`
```

Comment:
did you try `--overridesFile` instead?
https://github.com/swaggo/swag/#use-global-overrides-to-support-a-custom-type

apparently it would suffice with
`replace types.HexBytes string`
or maybe
`replace api/types.HexBytes string`

Reply:
(avoiding the need to specify `swaggertype:"string"`, shaving 38 changes off the PR)

Reply:
0ad7a1a5